src/services/goals.js

```diff
@@ -391,6 +392,7 @@ async function removeActivityReportObjectivesFromReport(reportId, objectiveIdsTo
   });
 }
 
+// TODO: ask Josh what the intent of this is for
```

Comment:
`removeUnusedGoalsObjectivesFromReport` calls this function to remove any goals on activity reports that are no longer in use. A user adds a goal to an AR, saves, and then removes that goal from the AR and saves again.